linux/build.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+# TODO: add keyman-config as child project
+
+cd "$THIS_SCRIPT_PATH"
+
+OUTPUT_PATH=
+
+builder_describe \
+  "Build Keyman for Linux." \
+  "clean" \
+  "configure" \
+  "build+" \
```

Comment:
```suggestion
  "build" \
```

Reply:
To be clear: https://github.com/keymanapp/keyman/blob/3c08d8f6b1bc310f042bcfd9b5abab1e64fc1226/resources/builder.inc.sh#L473-L475